### DISCUSSION TITLE
add a key for lables of x-result.json

### DISCUSSION THREAD

**Discussion by Gzxl** [allure-framework/allure2#1751] [Questions & Support]
I want to put case assert info store in the result.json from pyallure 
just like：
<img width="822" alt="image" src="https://user-images.githubusercontent.com/6359205/188631840-d1ac0408-0518-4e80-8d7c-a3f99dde43e2.png">
how to add a dynamic key for each case

**Comment by cheshi-mantu** [comment]
@Gzxl 
not sure if I understand what you are trying to do here. Could you please elaborate a bit?

btw, what are you trying to achieve using this?

<img width="732" alt="image" src="https://user-images.githubusercontent.com/8607348/188636332-7c175236-f642-4a3f-8508-a0d708dbb144.png">
